.github/workflows/docker_build_vttestserver.yml

```diff
@@ -21,7 +21,7 @@ jobs:
     strategy:
       fail-fast: true
       matrix:
-        branch: [ mysql57, mysql80 ]
+        branch: [ mysql80 ]
```

Comment:
I suppose no one has asked for a percona80 version of this image.

Reply:
Not that I am aware of, but we could easily add it if needed.